web/src/composebox_typeahead.ts

```diff
@@ -120,7 +124,6 @@ export let emoji_collection: Emoji[] = [];
 // get rid of it altogether.
 let completing: string | null;
 let token: string;
-
```

Comment:
Undo this removing of a line as part of the diff. A good thing to practice in the future if you look through your PR and see if there are any lines added/removed that aren't related to your PR.